toolkit/offchain/tests/integration_tests.rs

```diff
@@ -407,3 +437,26 @@ async fn assert_reserve_deposited<T: QueryLedgerState>(
 	)
 	.await;
 }
+
+async fn assert_mutable_settings_eq<T: QueryLedgerState + ogmios_client::OgmiosClient>(
+	genesis_utxo: UtxoId,
+	updated_mutable_settings: ReserveMutableSettings,
+	client: &T,
+) {
+	let ctx = TransactionContext::for_payment_key(GOVERNANCE_AUTHORITY_PAYMENT_KEY.0, client)
+		.await
+		.unwrap();
+	let reserve_data = crate::reserve::get_reserve_data(genesis_utxo, &ctx, client).await.unwrap();
```

Comment:
Is it possible to make this assertion using ReserveDatum and parsing OgmiosUtxo - it will be lower level, but I can see a lot things made public above in this PR, I think for this test.